src/main/java/com/powsybl/network/conversion/server/dto/EquipmentInfos.java

```diff
@@ -110,6 +110,24 @@ public static Set<VoltageLevel> getVoltageLevels(@NonNull Identifiable<?> identi
         throw NetworkConversionException.createEquipmentTypeUnknown(identifiable.getClass().getSimpleName());
     }
 
+    // TODO This function need to transfer to poswybl-ws-commons at the new release
+    public static String getEquipmentTypeName(@NonNull Identifiable<?> identifiable) {
+        return identifiable.getType() == IdentifiableType.HVDC_LINE ? getHvdcTypeName((HvdcLine) identifiable) : identifiable.getType().name();
+    }
+
+    /**
+     * @param hvdcLine The hvdc line to get hvdc type name
+     * @return The hvdc type name string
+     * @throws NetworkConversionException if converter station types don't match
+     */
+    private static String getHvdcTypeName(HvdcLine hvdcLine) {
+        if (hvdcLine.getConverterStation1().getHvdcType() != hvdcLine.getConverterStation2().getHvdcType()) {
+            throw NetworkConversionException.createHybridHvdcUnsupported(hvdcLine.getId());
+        }
+
+        return String.format("%s_%s", hvdcLine.getType(), hvdcLine.getConverterStation1().getHvdcType());
```

Comment:
```suggestion
        return String.format("%s_%s", hvdcLine.getType().name(), hvdcLine.getConverterStation1().getHvdcType().name());
```
toString() should be used only for label/log

Reply:
done